libpod/options.go

```diff
@@ -320,12 +320,18 @@ func (r *Runtime) WithPod(pod *Pod) CtrCreateOption {
 
 // WithLabels adds labels to the container
 func WithLabels(labels map[string]string) CtrCreateOption {
-	return ctrNotImplemented
-}
+	return func(ctr *Container) error {
+		if ctr.valid {
+			return ErrCtrFinalized
+		}
 
-// WithAnnotations adds annotations to the container
```

Comment:
Are we not implementing WithAnnotations?

Reply:
Annotations are stored directly in the spec. We already store the spec. No reason to store additional annotations elsewhere.